documentation/modules/overview/con-metrics-overview-tools.adoc

```diff
@@ -0,0 +1,63 @@
+// This module is included in:
+//
+// overview/assembly-metrics-overview.adoc
+
+[id="metrics-overview-tools-{context}"]
+= Prometheus
+
+Prometheus can extract metrics data from Kafka components and the Strimzi Operators.
+
+To use Prometheus to obtain metrics data and provide notifications, Prometheus and the Prometheus Alertmanager plugin must be deployed.
+Kafka resources must also be deployed or redeployed with Prometheus configuration to expose the metrics data.
+
+Prometheus scrapes the exposed metrics data for monitoring.
+Alertmanager issues notifications on conditions that indicate potential issues based on pre-defined alerting rules.
```

Comment:
it should be "alerts" not "notifications".
Then, not a native speaker but ... does the double "issues" (of course with different meanings) sound ok in English?

Reply:
Yeah, the Issue/issue isn't great, particulalrly as we're conveying two meanings.
Changed to:

> Alertmanager issues alerts when conditions indicate potential problems, based on pre-defined alerting rules.